<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CNBGeneratorBoundaryConditionsEditorWidget</class>
 <widget class="QFrame" name="CNBGeneratorBoundaryConditionsEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CNBGeneratorBoundaryConditionsEditorWidget</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblMinN">
     <property name="text">
      <string>Min N:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbMinN">
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>16777216</number>
     </property>
     <property name="singleStep">
      <number>1024</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblMaxN">
     <property name="text">
      <string>Max N:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="sbMaxN">
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>16777216</number>
     </property>
     <property name="singleStep">
      <number>1024</number>
     </property>
     <property name="value">
      <number>1048576</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMinTPB">
     <property name="text">
      <string>Min threads per block:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbMinTPB">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblMaxTPB">
     <property name="text">
      <string>Max threads per block:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sbMaxTPB">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1024</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1024</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
